docs/docs/using-mdx.mdx

```diff
@@ -41,7 +41,8 @@ That’s *roughly* turned into the following JavaScript.
 The below might help to form a mental model:
 
 ```tsx path="output-outline.jsx"
-/* @jsxRuntime automatic @jsxImportSource react */
+/*@jsxRuntime automatic*/
+/*@jsxImportSource react*/
```

Comment:
```suggestion
/* @jsxRuntime automatic */
/* @jsxImportSource react */
```

This one is not exact output, but a rough idea on how it looks, so can be pretty